overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -6598,7 +6598,7 @@
n- Keep relocating the Essentia Smeltery. Only break it when the purple bar on the left is EMPTY, otherwise you\u0027ll release a lot of Flux, probably spawning a Flux Rift in the process.\n\n- Rush to complete this Chapter and the very beginning of Bewitchment (the next Chapter), to gain access to the Flux Condenser and the Flux Condenser Lattice. Using this machine you\u0027ll be able to slowly convert Flux in a Chunk into Vitium Essentia. If you feed your Essentia Smeltery slowly, you can completely get rid of Flux in a Chunk this way, and you won\u0027t have to worry about it. This might require multiple Flux Condensers for a single Smeltery though.\n\n- Make a glass roof at build height in that chunk and around it or wait until Mortum and put your smelters there.\n\nTwo community assembled list of a group of preferred items to obtain Aspects, one intended for multiple essentia types per item and one intended for one essentia per item:\n\nFor the first one:\n\nPacked Ice - 15 §lGelum§r, 5 Ordo\nRotten Flesh - 5 §lHumanus§r, 5 §lVictus§r, 5 §lPerditio§r\nSkeleton Skull - 10 §lMortuus§r, 10 §lSpiritus§r, 10 §lExanimis§r\nEnder Arrow (Quark) - 11 §lMotus§r, 3 §lAversio§r, 7 §lAlienis§r\nAmber - 10 Vitreus, 10 §lVinculum§r\nOpal - 4 §lLuna§r, 4 Vitreus, 4 Praecantatio, 4 Desiderium\nGarnet - 4 §lStellae§r, 4 Vitreus, 4 §lPraemunio§r, 4 Desiderium\nScorned bricks - 10 §lDiabolus§r, 10 Praecantatio, 10 §lTenebrae§r, 10 §lIgnis§r\nDropper - 10 §lPermutatio§r, 26 Terra, 5 §lVacuos§r, 7 Potentia, 5 Machina\nCrafting Table - 12 §lInstrumentum§r, 20 §lFabrico§r, 12 Terra\nBook and Quill - 18 Herba, 3 Sensus, 11 §lAqua§r, 3 §lBestia§r, 3 §lVolatus§r, 3 §lCognitio§r, 10 §lAer§r\nPereskia - 5 Herba, 10 §lAuram§r, 2 Praecantatio, 2 Ordo\nEnergetic Alloy - 5 §lSol§r, 10 §lLux§r, 10 §lMetallum§r, 5 Sensus, 10 Potentia, 10 Desiderium\nNether Wart - 1 Herba, 2 §lVitium§r, 3 §lAlkimia§r\n\n(The §lbold§r aspects are unique to that item.)\n\nFor the second one:\n\nWool - Bestia(1)\nWool - Fabrico(1)\nNether wart - Vitium(2)\nAconitum - Mortus(1)\nAmber - Vinculum(1)\nLead - Ordo(1)\nEnder pearls - Motus(1)\nClay - Aqua(1)\nCopper - Permutatio(1)\nOak buttons - Machina\nEye of Ender - Praecantatio(1)\nIron ingot - Metallum\nBlaze powder - Alkimia(2)\nSkeleton skull - Exanimis(2)\nWooden Shovel - Instrumentum\nBottle o Enchanting - Cognitio\nPumpkin - herba\nDirt- Terra\nBone block - Sensus\nBlaze rod - Ignis(1)\nRedstone - Potentia\nTorches - Lux(1)\nFeathers - Aer(1)Feathers-  Volatus(1)\nEnder pearls- Alienis(1)\nBowl- Vacuos\nSilverwood log- Auram(1)Garnet - stellae(3)\nSnow - Gelum\nWooden sword - Aversio\nInvar boots - Praemunio\nGlass - Vitreus\nBlock of gold - Desiderium(1)\nGold ingot - Sol(2)\nHellbore seeds - Diabolus(1)\nRotten flesh- Humanus(2)\nOwl wings - Luna(2)\nSand - Perditio(1)\nWheat - Victus(1)\nWormwood seeds - Spiritus(1)\nDark gem - Tenebrae(1)\n\n(Each item has a (number) which signifies how many aspects the item has that you do not want.)"
```

Comment:
`Three` should not be capitalized. also, why specify Mortum? most dimensions with a bedrock roof works.

Reply:
I’ve heard that nether doesn’t, but should go test that